packages/remirror__extension-embed/src/iframe-extension.ts

```diff
@@ -4,33 +4,23 @@ import {
   command,
   CommandFunction,
   cx,
+  EditorView,
   extension,
   ExtensionTag,
   findSelectedNodeOfType,
   LiteralUnion,
   NodeExtension,
   NodeExtensionSpec,
   NodeSpecOverride,
+  NodeViewMethod,
   object,
   omitExtraAttributes,
   ProsemirrorAttributes,
+  ProsemirrorNode,
   Shape,
-  Static,
 } from '@remirror/core';
 
-export interface IframeOptions {
-  /**
-   * The default source to use for the iframe.
-   */
-  defaultSource?: Static<string>;
-
-  /**
-   * The class to add to the iframe.
-   *
-   * @default 'remirror-iframe'
-   */
-  class?: Static<string>;
-}
+import { IframeOptions, ResizableIframeView } from './resizable-iframe-view';
```

Comment:
It may be a good idea to keep `IframeOptions` in `iframe-extension.ts` since (1) that's what most extensions do, and (2)  ·IframeOptions· plays more of a role as options for the Extension than as options for the nodeView.